### DISCUSSION TITLE
How to specify more than one udev property in the configuration yaml file

### DISCUSSION THREAD

**Discussion by vermaete** [labgrid-project/labgrid#1338] [Q&A]
Hi,

The [documentation](https://labgrid.readthedocs.io/en/latest/configuration.html#androidusbfastboot) seems to give the possibility to have more that one udev property.

> match (dict): key and value pair**s** for a udev match

But I could not find an example how to do in the documentation.

My case is that we have now a new sets of FTDI USB to serial cables with another id.
Being able to specify both possible ID's would help.

br

**Comment by Bastian-Krause** [comment]
From the [udev Matching docs](https://labgrid.readthedocs.io/en/latest/configuration.html#udev-matching):
> To identify the kernel device which corresponds to a configured USBResource, each existing (and subsequently added) kernel device is matched against the configured resources. This is based on a list of match entries which must all be tested successfully against the potential kernel device.

**Reply by vermaete** [reply]
Well, you're right.  But having an example in the documentation would be helpful.

**Reply by Bastian-Krause** [reply]
What do you mean? There are resources with multiple match entries in the documentation, e.g. [USBFlashableDevice](https://labgrid.readthedocs.io/en/latest/configuration.html#usbflashabledevice). Note that your use case is not covered, though: *all* entries must match, not "one of them".
